File: app/src/main/java/com/neome/api/meta/base/dto/EntVdBotAddPromptToHistory.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdBotAddPromptToHistory extends EntVdAutoStep
{
  @Nullable
  public StudioDtoArgValueParameter documentAttachment;

  @Nullable
  public StudioDtoArgValueParameter imageAttachment;

  @Nullable
  public StudioBuildArgBinder label;

  @Nullable
  public StudioDtoArgValueParameter promptField;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdBotAddPromptToHistory.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStep
import com.neome.api.meta.base.dto.StudioBuildArgBinder
import com.neome.api.meta.base.dto.StudioDtoArgValueParameter

interface EntVdBotAddPromptToHistory : EntVdAutoStep
{
  val documentAttachment: StudioDtoArgValueParameter?
  val imageAttachment: StudioDtoArgValueParameter?
  val label: StudioBuildArgBinder?
  val promptField: StudioDtoArgValueParameter?
}